src/js/components/LeftPane.js

```diff
@@ -122,53 +135,82 @@ function InvestigationTree() {
 }
 
 export function LeftPane() {
+  const dispatch = useDispatch()
   const [showCollapse, setShowCollapse] = useState(true)
   const [showHistory, setShowHistory] = useState(true)
   const [showSpaces, setShowSpaces] = useState(true)
+  const [spacesHeight, setSpacesHeight] = useState(1)
+  const [historyHeight, setHistoryHeight] = useState(1)
   const view = useSelector(Layout.getInvestigationView)
   const isOpen = useSelector(Layout.getLeftSidebarIsOpen)
   const width = useSelector(Layout.getLeftSidebarWidth)
-  const dispatch = useDispatch()
   const id = useSelector(Tab.clusterId)
   const spaces = useSelector(Spaces.getSpaces(id))
   const spaceContextMenu = menu.spaceContextMenu(id)
 
-  function onDrag(e: MouseEvent) {
+  const spacesRef = useRef()
+  const historyRef = useRef()
+  const paneRef = useRef()
+
+  const paneHeight = useRef(0)
+
+  function onDragPane(e: MouseEvent) {
     const width = e.clientX
     const max = global.innerWidth
     dispatch(Layout.setLeftSidebarWidth(Math.min(width, max)))
   }
 
+  function onDragSpaces({dy, type}) {
```

Comment:
While we're dragging, well want to change the cursor of the mouse to remain in the "row-resize" position.

Right now as I drag, the cursor flickers between "row-resize" and "default" as I enter and leave the DragAnchor. 

To do this with javascript, you can either add class to the body called like "dragging-row" with the proper css.

Or you can use inline styles and set `document.body.style.cursor = "row-resize" in the "down" event, then switch back to `document.body.style.cursor = ""` in the "up" event.